frontend/src/ManageOrgUsers/ManageOrgUsers.jsx

```diff
@@ -220,7 +220,7 @@ class ManageOrgUsers extends React.Component {
             {!showNewUserForm && (
               <div className="container-xl">
                 <div className="card">
-                  <div className="card-table table-responsive table-bordered">
+                  <div className="card-table table-responsive table-bordered userTable">
```

Comment:
please use `user-table` instead of `userTable` to be consistent with other class names.